src/validate/mod.rs

```diff
@@ -23,21 +23,21 @@ lazy_static! {
     static ref AWS_ACCESS_FORMAT: Regex = Regex::new("^[A-Z0-9]+$").unwrap();
     static ref AWS_SECRET_FORMAT: Regex = Regex::new("^[A-Za-z0-9+/=]+$").unwrap();
     static ref BASE_URI_FORMAT: Regex = Regex::new(
-        "^https?://[A-Za-z0-9-]+(?:\\.[A-Za-z0-9-]+)*(?::[0-9]+)?/(?:[A-Za-z0-9-]+/)*$"
+        r"^https?://[A-Za-z0-9-]+(?:\.[A-Za-z0-9-]+)*(?::[0-9]+)?/(?:[A-Za-z0-9-]+/)*$"
```

Comment:
Oh cool, I didn't know regex literals are a thing!

Reply:
Yeah, it wouldn't let me do de `\pL` otherwise, said that `\p` wasn't a character they could escape.

Reply:
I changed all the other ones as well to avoid that problem and also all the examples in the `Regex` docs are like that (https://docs.rs/regex/1.0.2/regex/)